app/src/main/resources/openapi_etor.yaml

```diff
@@ -40,3 +49,10 @@ components:
         patientId:
           type: string
           example: MRN7465737865
+    UnauthorizedError:
+      description: Unauthorized Request
+  securitySchemes:
+    ApiKeyRequired:
+      type: apiKey
+      in: header
+      name: Authorization
```

Comment:
I read some of the other documentation for OpenAPI.  I would use a [bearer authentication](https://swagger.io/docs/specification/authentication/bearer-authentication/) instead.